templates/profile_view/profile.html

```diff
@@ -8,26 +8,56 @@
     <link rel="stylesheet" href="{% static 'profile_view/style.css' %}">
 </head>
 <body>
+    <header>
+        <nav>
+            <ul class="operation">
+                <li><a href="#">Explorer</a></li>
```

Comment:
can we make it so that the buttons change color slightly on hover to make it clear that they are buttons

Reply:
Sure